features/auth/useAuthStore.ts

```diff
@@ -70,19 +73,22 @@ export default function useAuthStore() {
       clearError() {
         setError(null);
       },
+      async resetAuthState() {
+        setAuthenticated(false);
+        setUserId(null);
+        setToken(null);
+        window.sessionStorage.clear();
+        Sentry.setUser({ id: undefined });
```

Comment:
```suggestion
        Sentry.setUser(null);
```
Sentry's docs seem to recommend setting to user to `null` instead, which makes more sense to me as I usually find having to explicitly set things to `undefined` a bit strange